regexes/device/mobiles.yml

```diff
@@ -14289,6 +14329,8 @@ Xiaomi:
       model: 'Redmi 10X Pro'
     - regex: 'Redmi K20 Pro(?:[);/ ]|$)'
       model: 'Redmi K20 Pro'
+    - regex: 'Redmi K30 5G(?:[);/ ]|$)'
+      model: 'Redmi K30 5G'
 
     - regex: 'MI PAD 4 PLUS(?:[);/ ]|$)'
       model: 'Mi Pad 4 Plus'
```

Comment:
should we make the same format for MI * model names for the Xiaomi brand?
`Mi Pad 4 Plus`  => `MI Pad 4 Plus`

Reply:
Sure. Sounds good